index.html

```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
                 <div class="lets-connect">
                     <h5>LET'S CONNECT</h5>
                 </div>
-
                 <div class="icons">
                     <ul>
                         <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
-                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img src="Pictures/Linkedin.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
+                                    src="Pictures/Linkedin.png" alt=""></a></li>
                         <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
                         <li><a href="https://github.com/Boku52500"><img src="Pictures/github.png" alt=""></a></li>
                         <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
                     </ul>
                 </div>
             </div>
-        </section>
+    </section>
+
+
+
+
+    <section class="2-page">
+        <div class="snapshot">
+            <a href=""><img src="Pictures/snapshot.png" alt=""></a>
+        </div>
+        <h2>Tonic</h2>
+        <div class="bullets">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+    <section class="3-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot2.png" alt=""></a>
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
```

Comment:
- [ ] Kindly try to follow the Figma design as much as possible, this includes card borders, the alignment of the cards (yours are to the left while in Figma they're centered), the buttons (background-color and interactions), the technologies badges also look very different from the ones on Figma.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192100498-206bf70f-f1bd-43ea-927b-a4c47e0637ca.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192100530-83abf547-d45b-4c99-887d-07f4af9f41c2.png)